src/main/webapp/dev/js/main/instructorCourseDetails.js

```diff
@@ -110,10 +111,6 @@ $(document).ready(() => {
     attachEventToDeleteStudentLink();
     attachEventToDeleteAllStudentLink();
 
-    $('#btn-select-element-contents').on('click', () => {
-        selectElementContents($('#detailsTable').get(0));
-    });
-
```

Comment:
Apologies; I showed the wrong code snippet on the original issue. It is lines 104-106 that should be deleted instead - can you revert this change and delete those lines? Have updated the issue.